package-lock.json

```diff
@@ -1,8 +1,1828 @@
 {
```

Comment:
Should I be concerned about this massive diff?

Reply:
I don't actually know! Maybe related to the installation of CableReady?